calculator_gem.rb

```diff
@@ -0,0 +1,47 @@
+require "colorize"
+
+puts "Let us do some simple maths! What kind of math do you want to do?"
+operation = gets.chomp
+
+# Checks for acceptable operators.
+correct_op_answers = ["+", "add", "-", "subtract", "*", "multiply", "/", "divide", "**", "exponent", "^", "%", "modulus"]
+
+correct_op_answers_colored = ["+".colorize(:blue), "add".colorize(:blue), "-".colorize(:cyan), "subtract".colorize(:cyan), "*".colorize(:yellow), "multiply".colorize(:yellow), "/".colorize(:green), "divide".colorize(:green), \
+"**".colorize(:magenta), "exponent".colorize(:magenta), "^".colorize(:magenta), "%".colorize(:red), "modulus".colorize(:red)]
+
+# Used a while loop to display the same error message until a valid operator is entered.
+while correct_op_answers.include?(operation) == false
```

Comment:
Nice! I like how this handles if someone tries to type in gibberish instead of one of your specified operators.